Zend/Zend.m4

```diff
@@ -206,9 +206,11 @@ else
   AC_DEFINE(ZEND_DEBUG,0,[ ])
 fi
 
-test -n "$GCC" && CFLAGS="-Wall -Wextra -Wno-strict-aliasing -Wno-implicit-fallthrough -Wno-unused-parameter -Wno-sign-compare $CFLAGS"
+test -n "$GCC" && CFLAGS="-Wall -Wextra -Wno-strict-aliasing -Wno-unused-parameter -Wno-sign-compare $CFLAGS"
 dnl Check if compiler supports -Wno-clobbered (only GCC)
 AX_CHECK_COMPILE_FLAG([-Wno-clobbered], CFLAGS="-Wno-clobbered $CFLAGS", , [-Werror])
+dnl Check for support for implicit fallthrough level 1, also add after previous CFLAGS as level 3 is enabled in -Wextra
+AX_CHECK_COMPILE_FLAG([-Wimplicit-fallthrough=5], CFLAGS="$CFLAGS -Wimplicit-fallthrough=5", , [-Werror])
```

Comment:
Should this be `=1`?

Reply:
Yes, I just wanted to make one last pass with the highest level to see if we had some more places in our own codebase where we could use the new keyword, turns out I missed one instance in PDO.

Should be ready now.